test/basic.dev.test.js

```diff
@@ -20,7 +20,10 @@ test.serial('Init Nuxt.js', async t => {
     dev: true,
     build: {
       stats: false,
-      profile: true
+      profile: true,
+      extractCSS: {
+        allChunks: true
```

Comment:
(Not related to this PR) If it is a better approach, what's your idea if we make default behavior of `extractCSS` to extract all chunks?

Reply:
About this, I have discussed with @Atinux several days ago https://github.com/nuxt/nuxt.js/pull/2081#issuecomment-343670353.  

I think bundle all css in aysnc component into a common css may be not good for performance.